forwards_paper.tex

```diff
@@ -526,20 +545,62 @@ \subsection*{Tree sequence simplification}
 (These lineages are the ``colors'' above.)
 We also maintain a map $A$ such that $A_j$ is segment chain for node $j$ in
 the input tree sequence.
-Crucially, the time required to run the algorithm is linear in the number of edges of the input tree sequence..
+Crucially, the time required to run the algorithm is linear in the number of edges of the input tree sequence.
+
+\subsection*{Forward simulation and sequential simplification to record simplifed tree sequences}
 
-\paragraph{Sequential simplification}
-At any point the simulation scheme above, genealogical history is recorded
-in a tree sequence. This has two additional advantages. First, simplification
+At any point in Algorithm~\algref{W}, genealogical history is recorded
+in an unsimplified tree sequence. This has two additional advantages. First, simplification
 can be run periodically through the simulation, taking the set of samples to be
 the entire currently alive population. This is important as it keeps memory
 usage from growing linearly (and quickly) with time. Second, the simulation can
 be \emph{begun} with a tree sequence produced by some other method -- for
 instance, by a coalescent simulation with \msprime. 
+This process of forward simulation coupled with sequential simplification is our
+method
+
+\jda{I didn't write down the full thing, but mightn't things be clearer if we
+	have a statement of the combined method?}
```

Comment:
Hm, good thought.  But I think my vote is to leave this out - it feels to me like algorithm W was clear enough.  If the reader understands (a) how alg W records tree seqeunces as nodes and edges, and (b) what simplification does, then putting them together is only a matter of juggling indices.

Reply:
True enough. I meant to add more justification. Here it is ---  I think there's two reasons to consider it:

1. it'll be clearer to the reader _what_ we're analyzing in the complexity analysis if we can say it's alg F with delta = 1 and the simulation with alg W. 
2. when we refer to our method or some such and say we implement it two ways it's clearer what we actually mean

Reply:
I'm not sure what you mean by (2)?

Reply:
I rephrased so maybe makes more sense. 

One example is the abstract https://github.com/petrelharp/ftprime_ms/blob/1a189620bb6770eba2a7dfb02ad12481b73b8b88/forwards_paper.tex#L50 (though we would not refer to the specific alg there. But say for the results figures: we are using a method that combines simulation with sequential simplification at some interval, i.e. F. Perhaps it doesn't need to be a listing, but I think it'd be nice to have this subsection leading into the complexity analysis and then the subsequent results where we specify this overall method of simulation + sequential simplification